tests/viz_tests.py

```diff
@@ -259,7 +259,7 @@ def test_adhoc_filters_overwrite_legacy_filters(self):
             query_obj['extras']['having_druid'],
         )
         self.assertEqual('(value3 in (\'North America\'))', query_obj['extras']['where'])
-        self.assertEqual('', query_obj['extras']['having'])
+        self.assertEqual('having': 'SUM(value1) > 5', query_obj['extras']['having'])
```

Comment:
@GabeLoins I'm unsure why `having` was supposed to be an empty string previously.